java-frontend/src/test/files/se/VarArgsYields.java

```diff
@@ -1,18 +1,20 @@
 class A {
-  private void varArgMethod(A... objects) {
```

Comment:
Seems that you are missing the case where the varargs part is empty. 

varArgMethod(A... objs) called without any arg.